template/public/app.js

```diff
@@ -1,34 +1,48 @@
-import moment from 'moment';
-import { uiModules } from 'ui/modules';
-import uiRoutes from 'ui/routes';
+import React from "react";
+import moment from "moment";
+import { uiModules } from "ui/modules";
+import chrome from "ui/chrome";
+import uiRoutes from "ui/routes";
+import { render, unmountComponentAtNode } from "react-dom";
 
-import 'ui/autoload/styles';
-import './less/main.less';
-import template from './templates/index.html';
+import "ui/autoload/styles";
+import "./less/main.less";
+import template from "./templates/index.html";
+import { Main } from "./components/main";
 
+const app = uiModules.get("apps/<%= name %>");
+app.config(stateManagementConfigProvider => {
+  stateManagementConfigProvider.disable();
+});
+const controller = ($scope, $route, $interval) => {
+  const elem = document.getElementById("pluginReactRoot");
+  let currentTime = moment($route.current.locals.currentTime);
+  const props = {
+    title: "<%= this.startCase(name) %>",
+    description: "<%= description %>",
+    currentTime: currentTime.format("HH:mm:ss")
+  };
+  render(<Main {...props} />, elem);
```

Comment:
Can we add a comment here:

```jsx
// Mount the React component tree on the DOM node.
render(<Main {...props} />, elem);
```

Reply:
Do we need this anymore? Can we just move all this into the react part of the app, and keep the angular bootstrapping really minimal? None of the props here require angular...

Removing the moment dependency from the template would also be great.